apps/coordinator/src/actions/transactionActions.js

```diff
@@ -529,7 +545,12 @@ export function importPSBT(psbtText, inputs, hasPendingInputs) {
     //
     // For now, we skip extraction to avoid breaking flows that lack full UTXO context.
     if (!hasPendingInputs) {
-      dispatch(setSignaturesFromPsbt(psbt));
+      try {
+        dispatch(setSignaturesFromPsbt(psbt));
+      } catch (sigError) {
```

Comment:
why should we not just be throwing the error and let the component error boundaries handle? If there's an error importing signatures, I think we want to let the user know about these very clearly and not continue on with the bad PSBT.